README.md

```diff
@@ -85,124 +78,198 @@ curl --location '{{ base-uri }}/resource' \
-for the schema of the request body.
-
+Resource can be updated by making a PUT request to the `/resource/{id}` endpoint. At present, only Work resources can be updated through the API.
+Refer [src/main/resources/swagger.api/schema/resourceRequestDto.json](https://github.com/folio-org/mod-linked-data/blob/master/src/main/resources/swagger.api/schema/resourceRequestDto.json) for the schema of the request body.
 ### Viewing a graph node
-A graph node can be viewed by making a GET request to the `graph/resource/{id}` endpoint. Any kind of resources can be
-retrieved through this API.
+A graph node can be viewed by making a GET request to the `graph/resource/{id}` endpoint. Any kind of resources can be retrieved through this API.
 
 ```bash
 curl --location '{{ base-uri }}/graph/resource/{id}' \
 --header 'x-okapi-tenant: {tenant}' \
 --header 'x-okapi-token: {token}'
 ```
-
-## Integration with FOLIO Inventory module
-When running in FOLIO mode, this module listens to MARC bibliographic records from the Kafka topic
-`{environment}.{tenantId}.DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING`. The received MARC records are
-converted to resource descriptions using the [lib-linked-data-marc4ld](https://github.com/FOLIO-EIS/lib-linked-data-marc4ld)
-library and then persisted in the graph.
-
-> **Note:** This process will be updated to integrate with [SRS](https://github.com/folio-org/mod-source-record-storage)
-> module in Ramsons release.
-
-## Integration with mod-search-ld module
-When running in FOLIO mode, this module will push the new and updated Work resource descriptions to the Kafka topic
-`{environment}.{tenantId}.search.bibframe` for consumption by [mod-search-ld](https://github.com/FOLIO-EIS/mod-search-ld)
-module. `mod-search-ld` will index the Work resource descriptions in [OpenSearch](https://aws.amazon.com/opensearch-service/)
-index for search and retrieval.
+# Integration with FOLIO
+When running in FOLIO mode, this module integrates with multiple Folio modules via Kafka.
+## Search module
+The Linked Data module pushes a new and updated Work resource descriptions to the Kafka topic defined by KAFKA_WORK_SEARCH_INDEX_TOPIC env variable.
+The [mod-search](https://github.com/folio-org/mod-search) module consumes message and indexes a Work resource descriptions in [OpenSearch](https://aws.amazon.com/opensearch-service/) index for search and retrieval.
+## Source Record Storage module
+The [mod-source-record-storage](https://github.com/folio-org/mod-source-record-storage) module pushes Source Record domain events to the topic defined by KAFKA_SOURCE_RECORD_DOMAIN_EVENT_TOPIC_PATTERN env variable.
+The Linked Data module consumes a message and creates/updates a corresponding Authority resource in Linked Data graph.
+## Inventory module
+1. The Linked Data module pushes a new and updated Instance resource descriptions to the Kafka topic defined by KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC_PATTERN env variable.
```

Comment:
I think the env variable should be KAFKA_INVENTORY_INSTANCE_INGRESS_EVENT_TOPIC.

`KAFKA_INVENTORY_INSTANCE_EVENT_TOPIC_PATTERN` is used for the suppression flags (as mentioned in line 274)